include/common_package/node_names.hpp

```diff
@@ -0,0 +1,34 @@
+//
+// Created by Johan <job8197@thi.de> on 17.05.2024.
+//
+
+#ifndef THI_DRONE_WS_NODE_NAMES_HPP
+#define THI_DRONE_WS_NODE_NAMES_HPP
+
+#include <set>
+
+namespace common_lib::node_names {
+
+constexpr char const *const MISSION_CONTROL =
+    "mission_control"; /**< Node name for the mission control node */
+
+constexpr char const *const WAYPOINT =
+    "waypoint"; /**< Node name for the navigate to waypoint node */
+
+constexpr char const *const QRCODE_DETECTION =
```

Comment:
```suggestion
constexpr char const *const QR_CODE_SCANNER_NODE =
```

Reply:
I left the Node out of the name because they are defined in the node_names namespace and this way it stays consistent with the style chosen in topic_names

Reply:
?

Reply:
The topic names are defined as BatteryState etc. and not as BatteryStateTopic or similar, so I opted to do the same here.

Reply:
Ok, I see